ComputerArchitecture/Ywoosang/2025-08-22-Ywoosang-CPU_메모리_구조_Overview.md

```
@@ -0,0 +1,327 @@
+- RAM에 빈 공간이 없는데 새로운 페이지가 필요하면, 기존에 있던 페이지를 내보내야 함 => 페이지 교체 
+- 교체할 페이지 선택은 알고리즘에 따라 결정
+  - LRU (Least Recently Used): 가장 오래 쓰이지 않은 페이지 제거
+  - FIFO (First In First Out): 가장 먼저 올라온 페이지 제거
+  - LFU (Least Frequently Used): 사용 빈도가 가장 낮은 페이지 제거
+
+
+## OS 에서 메모리 관리
+
+OS는 컴퓨터 하드웨어를 관리하는 소프트웨어임
+
+* CPU가 데이터를 요청했는데 메모리에 없으면 디스크까지 접근해야 하므로 성능이 크게 저하됨
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)